FSharp.AutoComplete/Program.fs

```diff
@@ -472,7 +476,7 @@ module internal Main =
                                  Data = { Project = file
                                           Files = files
                                           Output = targetFilename
-                                          References = List.sort p.References
+                                          References = List.sortBy Path.GetFileName p.References
```

Comment:
Sort by file name only, fixes the issue with references appearing out of order on my machine, sensitive to user names, possibly machine name, and other paths(?)